test/LidoStakedEthRewardsIntegration.t.sol

```diff
@@ -0,0 +1,194 @@
+            asset:             _getAToken(WETH),  // Rewards on WETH supplies
+            reward:            WSTETH,
+            transferStrategy:  transferStrategy,
+            rewardOracle:      IEACAggregatorProxy(aaveOracle.getSourceOfAsset(WSTETH))
+        });
+
+        vm.prank(operator);
+        emissionManager.configureAssets(configs);
+    }
+
+    function test_setup_distribution() public {
+        address wethAToken = _getAToken(WETH);
+
+        (
+            uint256 index,
+            uint256 emissionPerSecond,
+            uint256 lastUpdateTimestamp,
+            uint256 distributionEnd
+        ) = incentives.getRewardsData(wethAToken, WSTETH);
+        assertEq(index,                                  0);
+        assertEq(emissionPerSecond,                      0);
+        assertEq(lastUpdateTimestamp,                    0);
+        assertEq(distributionEnd,                        0);
+        assertEq(incentives.getTransferStrategy(WSTETH), address(0));
+        assertEq(incentives.getRewardOracle(WSTETH),     address(0));
+
+
+        _setupDistribution();
+
+        (
+            index,
+            emissionPerSecond,
+            lastUpdateTimestamp,
+            distributionEnd
+        ) = incentives.getRewardsData(wethAToken, WSTETH);
+        assertEq(index,                                  0);
+        assertEq(emissionPerSecond,                      REWARD_AMOUNT / DURATION);
+        assertEq(lastUpdateTimestamp,                    block.timestamp);
+        assertEq(distributionEnd,                        block.timestamp + DURATION);
+        assertEq(incentives.getTransferStrategy(WSTETH), address(transferStrategy));
+        assertEq(incentives.getRewardOracle(WSTETH),     address(aaveOracle.getSourceOfAsset(WSTETH)));
+    }
+
+    function test_user_claim() public {
+        address claimAddress = makeAddr("claimAddress");
+        address[] memory assets = new address[](1);
+        assets[0] = _getAToken(WETH);
+
+        vm.prank(whale1);
+        incentives.claimAllRewards(assets, claimAddress);
+        assertEq(IERC20(WSTETH).balanceOf(claimAddress), 0);
+
+        _setupDistribution();
+
+        vm.prank(whale1);
+        incentives.claimAllRewards(assets, claimAddress);
+        assertEq(IERC20(WSTETH).balanceOf(claimAddress), 0);
+
+        uint256 skipAmount = DURATION / 2;  // 50% of rewards distributed
+        skip(skipAmount);
+
+        // 7-siblings wallet should get about half of the rewards at this time
+        // 79k ETH deposit out of 157k total supplied
+        vm.prank(whale1);
+        incentives.claimAllRewards(assets, claimAddress);
+        uint256 whaleReward1 = 5.028734135612479150 ether;
+        assertEq(IERC20(WSTETH).balanceOf(claimAddress), whaleReward1);
+        assertEq(IERC20(WSTETH).balanceOf(operator),     REWARD_AMOUNT - whaleReward1);
+
+        skip(DURATION - skipAmount);
+
+        // 7-siblings wallet should get about 50% of the total rewards ~10 wstETH
```

Comment:
Not sure if we need to refer to these wallets in so non-anon way. Why not just whale 1? Also in this case the comment would match the variable naming below.

Reply:
fixed